Components/grid/grid.css

```diff
@@ -0,0 +1,125 @@
+@import url("../../main.css");
+@import url('../../colors.css');
```

Comment:
You have `colors.css` so why are you specifying more colors in this file? These could also be mentioned in `colors.css` and used here. It's just maintaining consistency and keeping all the colors code together.

Reply:
these colours are specific to that component only so i wanted to keep them separately to avoid confusion

Reply:
I have also kept consistency in mind so these separate colors are related to the main colors but with different shades, this is giving a better look to the home page and the main colors are intended to be used once the user enters the application.